### PR TITLE
Update presentations.md

### PR DESCRIPTION
**Pull request by helenlly**
update 3 talks in Flink Forward Asia 2019 and InfoQ

### CODE REVIEW

File: docs/docs/presentations.md [hunk 1]
```diff
@@ -4,8 +4,13 @@
 
 **Talks:**
 
-* Building AI to play the FIFA video game using distributed TensorFlow, [TensorFlow World](https://conferences.oreilly.com/tensorflow/tf-ca/public/schedule/detail/78309), Oct 2019, Santa Clara ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Building%20AI%20to%20play%20the%20FIFA%20video%20game%20using%20distributed%20TensorFlow.pdf))
+* Architecture and practice of big data analysis and deep learning model inference using Analytics Zoo on Flink(FFA2019), [Flink Forward Asia 2019](https://developer.aliyun.com/special/ffa2019-conference?spm=a2c6h.13239638.0.0.21f27955PCNMUB#), Nov 2019, Beijing ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Architecture%20and%20practice%20of%20big%20data%20analysis%20and%20deep%20learning%20model%20inference%20using%20Analytics%20Zoo%20on%20Flink(FFA2019)%20.pdf))
```
Comment (hkvision) [inline docs/docs/presentations.md]: Add a space before (FFA...

Comment (hkvision) [reply docs/docs/presentations.md]: Also missing space before (slides) in Tutorials part

Comment (jason-dai) [reply docs/docs/presentations.md]: remove FFA

File: docs/docs/presentations.md [hunk 2]
```diff
@@ -4,8 +4,13 @@
...
+
+* Data analysis + AI platform technology and case studies, [InfoQ AICon BJ 2019](https://qcon.infoq.cn/2019/beijing/), Nov 2019, Beijing ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/AICON%20AZ%20Cluster%20Serving%20Beijing%20Qiyuan_v5.pdf))
 
+* Architectural practices for building a unified big data AI application with Analytics-Zoo, [InfoQ QCon SH 2019](https://qcon.infoq.cn/2019/shanghai/presentation/1921), Oct 2019, Shanghai ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Architectural%20practices%20for%20building%20a%20unified%20big%20data%20AI%20application%20with%20Analytics-Zoo.pdf))
```
Comment (jason-dai) [inline docs/docs/presentations.md]: Just "QCon Shanghai"

File: docs/docs/presentations.md [hunk 3]
```diff
@@ -4,8 +4,13 @@
...
 
-* Building AI to play the FIFA video game using distributed TensorFlow, [TensorFlow World](https://conferences.oreilly.com/tensorflow/tf-ca/public/schedule/detail/78309), Oct 2019, Santa Clara ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Building%20AI%20to%20play%20the%20FIFA%20video%20game%20using%20distributed%20TensorFlow.pdf))
+* Architecture and practice of big data analysis and deep learning model inference using Analytics Zoo on Flink(FFA2019), [Flink Forward Asia 2019](https://developer.aliyun.com/special/ffa2019-conference?spm=a2c6h.13239638.0.0.21f27955PCNMUB#), Nov 2019, Beijing ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/Architecture%20and%20practice%20of%20big%20data%20analysis%20and%20deep%20learning%20model%20inference%20using%20Analytics%20Zoo%20on%20Flink(FFA2019)%20.pdf))
+
+* Data analysis + AI platform technology and case studies, [InfoQ AICon BJ 2019](https://qcon.infoq.cn/2019/beijing/), Nov 2019, Beijing ([slides](https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/presentations/AICON%20AZ%20Cluster%20Serving%20Beijing%20Qiyuan_v5.pdf))
```
Comment (jason-dai) [inline docs/docs/presentations.md]: the link is for QCon Bejjing, not AICon

File: docs/docs/presentations.md (hunk 1 above)
Comment (helenlly) [reply docs/docs/presentations.md]: done

File: docs/docs/presentations.md (hunk 2 above)
Comment (helenlly) [reply docs/docs/presentations.md]: done

File: docs/docs/presentations.md (hunk 3 above)
Comment (helenlly) [reply docs/docs/presentations.md]: done

**Comment by hkvision** [comment]
LGTM

**Pull request merged**